### PR TITLE
fix(@spartacus/storefront): Item is added to cart when manually change the Qty value to 0 or greater than max #499

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
#499

### CODE REVIEW

File: projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.spec.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ const testData = [
   { incomingValue: 6, adjustedValue: 5, isOutOfRange: true }
 ];
 
-describe('ItemCounterComponent', () => {
+fdescribe('ItemCounterComponent', () => {
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.spec.ts]: Remove this

File: projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.spec.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ const testData = [
...
   let itemCounterComponent: ItemCounterComponent;
   let fixture: ComponentFixture<ItemCounterComponent>;
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.spec.ts]: Cant write this anywhere else. Add a test to make sure something happens (warning) id the value is invalid.

File: projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.ts [hunk 3]
```diff
@@ -139,14 +132,21 @@ export class ItemCounterComponent
     }
   }
 
-  onBlur(event: FocusEvent) {
+  onInput(event: Event): void {
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.ts]: When is this used?

Comment (bgambocjaviniar) [reply projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.ts]: @LTiger14 Removed it! It was kept before when I merged develop to this branch

**Pull request merged**